Author: Boris Pek <tehnick-8@yandex.ru>
Last-Update: 2018-03-13
Description: define SQLITE_ENABLE_COLUMN_METADATA
 for including of sqlite3_column_table_name16 function into SQLite API which
 is used by latest versions of Qt 5.x and is required for building of qtbase
 package in a shared libraries mode.

diff --git a/src/sqlite.h.in b/src/sqlite.h.in
index a434493..a9a2adc 100644
--- a/src/sqlite.h.in
+++ b/src/sqlite.h.in
@@ -67,6 +67,12 @@
 # define SQLITE_SYSAPI
 #endif
 
+/*
+** Define SQLITE_ENABLE_COLUMN_METADATA for enabling of extra functions
+** in SQLite API.
+*/
+#define SQLITE_ENABLE_COLUMN_METADATA
+
 /*
 ** These no-op macros are used in front of interfaces to mark those
 ** interfaces as either deprecated or experimental.  New applications

